constructorio-client/src/main/java/io/constructor/client/ConstructorIO.java

```diff
@@ -88,9 +91,25 @@ public ConstructorIO(String apiToken, String apiKey, boolean isHTTPS, String hos
         } else {
             this.protocol = "http";
         }
+        if (constructorToken != null) {
```

Comment:
I believe the check is redundant. If it's null, you can set it to null which is the same as not setting it.

Reply:
Ah.  yay.